.travis.yml

```diff
@@ -28,13 +28,14 @@ before_install:
 - cabal --version
 
 install:
-- cabal new-update
+- cabal new-update hackage.haskell.org,HEAD
```

Comment:
What’s the `HEAD` do?

Reply:
Made sure we scraped from the absolute latest snapshot. I think we can kill this.